microsoft-edge/devtools-guide-chromium/contact.md

```diff
@@ -0,0 +1,31 @@
+---  
+description: Contacting the Microsoft Edge DevTools team.
+title: Contacting the Microsoft Edge DevTools team
+author: MSEdgeTeam  
+ms.author: msedgedevrel  
+ms.date: 08/06/2021  
+ms.topic: conceptual  
+ms.prod: microsoft-edge  
+ms.technology: devtools  
+keywords: microsoft edge, web development, f12 tools, devtools  
+---  
+# Contacting the Microsoft Edge DevTools team
+
+Send your feedback to help improve Microsoft Edge DevTools.  There are several ways to get in touch with the Microsoft Edge DevTools team.
+
+*  To send feedback about Microsoft Edge DevTools or the Microsoft Edge Developer documentation, select the **Send Feedback** icon in DevTools.  Or, in DevTools, select `Alt`+`Shift`+`I` (Windows, Linux) or `Option`+`Shift`+`I` (macOS).
+
+:::image type="complex" source="media/devtools-send-feedback.msft.png" alt-text="The Send Feedback icon in Microsoft Edge DevTools" lightbox="media/devtools-send-feedback.msft.png":::
+   The **Send Feedback** icon in Microsoft Edge DevTools  
+:::image-end:::  
+
+*  Tweet at [@EdgeDevTools][PostTweetEdgeDevTools].  
+
+*  Submit a suggestion through [The Web We Want initiative](../web-we-want/index.md).
```

Comment:
need a few-word description/hint about what this is, why you'd choose this communication channel, give guidance.  When/why should you use this approach?